app/models/district.js

```diff
@@ -6,11 +6,11 @@ import numeral from 'numeral';
 import neighborhoodsCrosswalk from '../utils/nabesCrosswalk';
 
 const acronymCrosswalk = {
-  Bronx: 'BX',
-  Brooklyn: 'BK',
-  Manhattan: 'MN',
-  Queens: 'QN',
-  'Staten Island': 'SI',
+  Bronx: 'X',
```

Comment:
What's the reasoning for this?

Reply:
dataset column values changed to single  characters.

Reply:
The `districts.js` has functions that take the single character and two digit community district and the screenshot demos the result. Without this change, the fetch fetches nothing :( 

<img width="346" alt="Screen Shot 2022-08-30 at 3 10 11 PM" src="https://user-images.githubusercontent.com/11340947/187523244-1e71b898-3893-450c-8c40-8409e9a75522.png">